### PR TITLE
Equip class2

### PR DESCRIPTION
**Pull request by nyee**
I fixed up the other pull request a little too fast. Can @abhishekpradhan or @MechE99 review this one?

### CODE REVIEW

File: src/Constants/Avail.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+"""Enum describing how availability of equipment."""
+
+from enum import Enum
+
+
+class Avail(Enum):
```
Comment (abhishekpradhan) [inline src/Constants/Avail.py]: Why not change this to Availability?

File: test/Equipment/MeleeWeapon_test.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+"""Unit tests for MelWeapons class."""
+
+import unittest
+from src.Constants.Avail import Avail
+from src.Constants.Elem import Elem
+from src.Constants.EquipType import EquipType
+from src.Equipment.MeleeWeapon import MeleeWeapon
+
+
+class Test_MelWeapons(unittest.TestCase):
```
Comment (abhishekpradhan) [inline test/Equipment/MeleeWeapon_test.py]: Test_MeleeWeapons

File: src/Constants/__init__.py [hunk 3]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (abhishekpradhan) [inline src/Constants/__init__.py]: don't need this blank line

**Comment by nyee** [comment]
I addressed your comments Ab, if you think it's good, let me fixup the commits before we pull in.

File: src/Constants/__init__.py (hunk 3 above)
Comment (nyee) [reply src/Constants/__init__.py]: Apparently we do, or we don't past flake8.

**Pull request merged**